src/lib/core/a11y/list-key-manager.ts

```diff
@@ -42,15 +42,19 @@ export class ListKeyManager {
   onKeydown(event: KeyboardEvent): void {
     switch (event.keyCode) {
       case DOWN_ARROW:
+        event.preventDefault();
         this.focusNextItem();
         break;
       case UP_ARROW:
+        event.preventDefault();
         this.focusPreviousItem();
         break;
       case HOME:
+        event.preventDefault();
         this.focusFirstItem();
         break;
       case END:
+        event.preventDefault();
```

Comment:
Instead of calling `preventDefault` in each case, you can do:
```ts
  onKeydown(event: KeyboardEvent): void {
    switch (event.keyCode) {
      case DOWN_ARROW:
        this.focusNextItem();
        break;
      case UP_ARROW:
        this.focusPreviousItem();
        break;
      case HOME:
        this.focusFirstItem();
        break;
      case END:
        this.focusLastItem();
        break;
      case TAB:
        this._tabOut.next(null);
        return;
      default: 
        return;
    }

    event.preventDefault();
  }
```
(see `slider.ts` for an example)